src/rulesets/__tests__/path.unit.ts

```diff
@@ -1,15 +1,16 @@
+import * as path from 'path';
 import { resolvePath } from '../path';
 
 describe('Path', () => {
-  it('should resolve relative path', () => {
-    expect(resolvePath('a/b/c', 'd')).toEqual(`${process.cwd()}/a/b/d`);
+  it('should join relative path', () => {
+    expect(resolvePath('a/b/c', 'd')).toEqual(`a/b/c/d`);
   });
 
-  it('should resolve absolute path', () => {
+  it('should join absolute path', () => {
     expect(resolvePath('a/b/c', '/d')).toEqual('/d');
   });
 
-  it('should resolve http relative', () => {
+  it('should join http relative', () => {
     expect(resolvePath('http://www.example.com/a/b.json', 'd.json')).toEqual('http://www.example.com/a/d.json');
```

Comment:
This is the only broken test, should be fixed once `@stoplight/path` is released.